providers/apis/defi/raydium/types.go

```diff
@@ -18,6 +18,25 @@ const (
 	NormalizedTokenAmountExponent = 18
 )
 
+// updateMetaDataCache unmarshals the metadata JSON for each ticker and adds it to the
+// metadata map.
+func (pf *APIPriceFetcher) updateMetaDataCache(ticker types.ProviderTicker) (TickerMetadata, error) {
+	if metadata, ok := pf.metaDataPerTicker[ticker.GetJSON()]; ok {
```

Comment:
The primary key is here is the json field, @nivasan1 do you think thats fine or should we just use the offchain ticker?